demos/common/python/requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy>=1.16.6,<=1.23.1
 opencv-python
-scipy~=1.5.4
+scipy~=1.7.3; python_version == '3.7'
```

Comment:
instead of pining specific version can we relax dependency, just >= 1.5.4?

Reply:
We had a discussion about it, would current dependency be alright? `scipy>=1.5.4,<1.9.0`

Reply:
yes, it is ok